definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -0,0 +1,34 @@
+# This list of inputs will replace any occurrence of `{Energy Storage System Type}`
+# in a variable-definition yaml file
+
+# This list is intended for use as input energy storage system to the power sector
+
+- Electricity Storage Type:
+
+  - Energy Storage System:
+      description: Mainly to refer to the batteries
+  - Energy Storage System|Lithium-Ion:
+      description: Lithium-Ion batteries
+  - Energy Storage System|Redox Flow:
+      description: Redox Flow batteries
+  - Energy Storage System|Compressed Air:
+      description: Compressed Air batteries
```

Comment:
Is compressed air really a battery?

Reply:
It was inherited from a PR made by Thorsten. @loeffko , Is it needed in GENeSYS-Mod?

Reply:
it is in the list of technologies of the scenarios, so for me we need to keep it., and yes it is an existing technology of stationnary battery

Reply:
Sorry for not being clear in my comment, I simply meant the description as "Compressed Air batteries" - I think "Energy storage using compressed air" would be more logical.